README.md

```diff
@@ -154,7 +154,7 @@ Cutting a new version refers to copying the full `/docs` directory contents to a
 
 To cut a new version:
 
-1. Run `npm run docusaurus docs:version version-<version>`, where `<version>` is an integer that corresponds to a major Helm version. For example, if you want to publish new pre-release docs for Helm v5, then you can run `npm run docusaurus docs:version version-4` to copy the v4 docs content to a new `version-4` folder.
+1. Run `yarn docusaurus docs:version version-<version>`, where `<version>` is an integer that corresponds to a major Helm version. For example, if you want to publish new pre-release docs for Helm v5, then you can run `yarn docusaurus docs:version version-4` to copy the v4 docs content to a new `version-4` folder.
```

Comment:
should be `yarn docusaurus docs:version 4` (not `version-4`). Also removed the part that repeats what you say the command does just below.

```suggestion
1. Run `yarn docusaurus docs:version <version>`, where `<version>` is an integer that corresponds to a major Helm version. For example, when we publish Helm 4.0.0 we will run `yarn docusaurus docs:version 4`.
```